pyk/src/pyk/k2lean4/k2lean4.py

```diff
@@ -113,6 +158,55 @@ def _collection(self, sort: str) -> Inductive:
         ctor = Ctor('mk', Signature((ExplBinder(('coll',), val),), Term(sort)))
         return Inductive(sort, Signature((), Term('Type')), ctors=(ctor,))
```

Comment:
Could we have `CellMap`s be `structure`s as well? So that, for instance
```lean
inductive SortMessageCellMap : Type where
    | mk (coll : List (SortMsgIDCell × SortMessageCell)) : SortMessageCellMap
```
becomes
```lean
structure SortMessageCellMap : Type where
    coll : List (SortMsgIDCell × SortMessageCell)
```

Reply:
Well, all collections really, since they only have one constructor

Reply:
Done.